CONTRIBUTING.md

```diff
@@ -271,6 +271,36 @@ This allows:
 * the reviewer to be very confident that the feature does what it is supposed to do before merging it into the code base.
 * the contributors to be sure that they don't break already-merged features when refactoring or modifying the code base.
 
+<a name="-enable-logging"></a>
+## Enable logging
+See more logs by setting the log level to `DEBUG`.
+
+Example:
+```python
+import ...
+# import logging and set the level to DEBUG
+import logging
+logging.getLogger('docarray').setLevel(logging.DEBUG)
```

Comment:
Not sure if it should be in the contributing file.

I think the part that says to the user how to enable logging should not be in the contributing guideline but rather in the readme for now ( we need to build documentation later anyway)

THough we might need something in the contributing guideline that say: if you need to do some logging you can use docarray logger

Reply:
ok makes sense, logs are not for contributors only